packages/utilities/web-vitals/src/index.js

```diff
@@ -117,9 +120,9 @@ const useWebVitals = ({
       numberOfLogicalProcessors,
       deviceMemory,
     });
-    getCLS(updateWebVitals);
+    getCLS(updateWebVitals, true); // Setting 'true' will report all CLS changes
```

Comment:
Do we have / need tests for this?

Reply:
there's tests already in place to ensure this function is called. Not 100% sure we need to update the tests for this though

Reply:
Could we update the test make sure it has been called with `true` as a param?

Reply:
I'll do that in a separate PR as it involves mocking the updateWebVitals function which currently isnt done yet